### ISSUE TITLE
Question about results in Table 1

### ISSUE THREAD

**Issue by iamseungyun** [Y-debug-sys/Diffusion-TS#22]
Hi. I'm seungyun Lee.

I have a question about the results in Table 1.

Now, I'm in the process of replicating your experiment, but the results seem different. Is it possible that it's due to some hyperparameter issue, or is it simply an error in the experimental setup?

This is my result for unconditional generation using mujoco data.

![image](https://github.com/Y-debug-sys/Diffusion-TS/assets/139734174/9a5abf01-13d9-4c12-ae4e-5e7921b835f5)

Not just for mujoco, unconditional generation for other datasets and imputation also have little bit different results.

I'll waiting your reply. Thank you.

**Comment by Y-debug-sys** [comment]
Hi, seungyun. 

This may be due to the randomness in these experiments, where each sampling and calculation of metrics may result in slightly different outcomes. And I noticed your results are more unstable than it in the paper, so please try more times. Regarding etth and energy imputation, we used the first 90% for training, leaving the rest for the test set, but it's random divided in the repo. 

Thanks!
